src/docs/sphinx/t_conduit_docs_blueprint_demos_out.txt

```diff
@@ -31,7 +31,7 @@ fields:
 END_EXAMPLE("blueprint_demo_basic_uniform")
 [/Users/harrison37/Work/github/llnl/conduit/src/tests/docs/t_conduit_docs_blueprint_demos.cpp : 67]
  Testing Basic Example 'uniform'
-[       OK ] conduit_docs.blueprint_demo_basic_uniform (12 ms)
+[       OK ] conduit_docs.blueprint_demo_basic_uniform (10 ms)
```

Comment:
For the future, would it be possible to omit the timings for these test cases so that they aren't modified each time we update the example annotations in the documentation?